### PR TITLE
Change Computer get method to return all attributes instead of just DN

### PR DESCRIPTION
**Pull request by tenyo**

### CODE REVIEW

**Comment by fishnix** [comment]
As we talked about, I think you should update the `get` method instead of creating a new `get_all` method.  🥇

File: lib/ad_management/objects/computer.rb [hunk 1]
```diff
@@ -100,6 +91,7 @@ def move(cn, target_ou)
       private
 
       def to_hash(entry)
+        return '' if entry.nil?
```
Comment (fishnix) [inline lib/ad_management/objects/computer.rb]: `{}`?

**Comment by fishnix** [comment]
`to_h` should probably not return an empty _string_, otherwise looks good 👍

File: lib/ad_management/objects/computer.rb (hunk 1 above)
Comment (tenyo) [reply lib/ad_management/objects/computer.rb]: Good catch 🎣

**Pull request merged**